### ISSUE TITLE
can't get completions without having an accessible containerd socket

### ISSUE THREAD

**Issue by 06kellyjac** [AkihiroSuda/nerdctl#101]
When packaging programs it can be common to extract completions to a file to be put in `/usr/share/bash-completions/completions/x` (or similar) but can't be done without a containerd socket that's accessible

```bash
$ nerdctl completion bash
FATA[0000] rootless containerd not running? (hint: use `containerd-rootless-setuptool.sh install` to start rootless containerd): stat /run/user/1000/containerd-rootless: no such file or directory
```

This is also a problem for `nerdctl version` but that is more understandable since it'll output the containerd details

**Comment by AkihiroSuda** [comment]
Thanks for reporting, fixed in #102

**Comment by AkihiroSuda** [comment]
> This is also a problem for nerdctl version but that is more understandable since it'll output the containerd details

`nerdctl --version` shows client version without the daemon running
